crt_portal/crt_portal/settings.py

```diff
@@ -378,8 +379,10 @@
 if environment == 'LOCAL':
     from .local_settings import *  # noqa: F401,F403
 
+IS_RUNNING_TEST_SUITE = (os.path.basename(sys.argv[0]) == 'manage.py' and len(sys.argv) > 1 and sys.argv[1] == 'test')
```

Comment:
Hmm. This might not be needed. We already have similar (sigh, admittedly hacky) functionality for detecting environments. Not sure if this will work with CI, but maybe something like the following would work:

```python
if DEBUG and environment not in ['UNDEFINED', 'LOCAL']: ...
```

(Note: this approach is used elsewhere in this settings file -- I'm not a fan but if it works, let's strive for consistency?)

Reply:
Otherwise I would advocate for a separate setting configuration (if it's not too much work).

Reply:
Checking against the environment doesn't really help us in this case, if I understand things correctly, because at bottom, the issue is that...

https://github.com/usdoj-crt/crt-portal/blob/5dda152a2a1f8b79a5f84a1202aae5980486b57b/crt_portal/crt_portal/urls.py#L90-L93

... will resolve to `False` when running `manage.py test`, and `True` when running `manage.py runserver`, all other things being equal. 

That said, after some more thought I also think it would be cleaner to capture this toggling as its own configuration, so I'll push that change momentarily. Only thing to keep in mind with this approach is that the toolbar will (by default) be enabled for local test runs, which maybe isn't a big deal

Reply:
Ahh I see now. Yes, thank you for the explanation -- I wasn't looking closely enough at this issue. What you have LGTM.